bindings/java/c/host.c

```diff
@@ -66,6 +68,7 @@ static evmc_bytes32 get_storage_fn(struct evmc_host_context* context,
     evmc_bytes32 result;
     const char java_method_name[] = "get_storage";
     const char java_method_signature[] = "(I[B[B)Ljava/nio/ByteBuffer;";
+    assert(context != NULL);
```

Comment:
There are makefile targets: `build` and `debug`. I'm not sure this is intentional, but asserts are enabled in both. I think this is good at least at this phase of development.

Reply:
I think it is intentional currently.

Reply:
It will not be when we move it under CMake jurisdiction.

Reply:
What do you suggest to use?

Reply:
Looking at the extent of changes I have a preference to merge this first and then the cmake PR also replacing the asserts.

Reply:
`assert` is fine.